Foundation/CGFloat.swift

```diff
@@ -143,6 +143,10 @@ public struct CGFloat {
     
     /// The native value.
     public var native: NativeType
+
+    @_transparent public var double: Double {
+        return Double(native)
+    }
```

Comment:
Reflecting on this it seems people expect a cast to Double at the call sites. I’ll do this.